TidepoolKit Example/Support/Extensions/Sample.swift

```diff
@@ -21,6 +21,23 @@ struct Sample {
                 TCBGDatum(time: Date(),
                           value: 123,
                           units: .milligramsPerDeciliter),
+                TCGMSettingsDatum(time: Date(),
+                                  manufacturers: ["Acme Pump", "Acme CGM"],
+                                  model: "Super Deluxe",
+                                  serialNumber: "999666333",
+                                  transmitterId: "A1B2C3",
+                                  units: .milligramsPerDeciliter,
+                                  defaultAlerts: cgmSettingsAlerts,
+                                  scheduledAlerts: [TCGMSettingsDatum.ScheduledAlert(name: "Default",
+                                                                                     days: ["sunday", "monday", "tuesday", "wednesday", "thursday", "friday", "saturday"],
+                                                                                     start: 0,
+                                                                                     end: 64800000,
```

Comment:
Is this is seconds or milliseconds? Either way is `64800000` a good value?

Reply:
Per backend documentation, this is milliseconds and can be up to 24 hours. The maximum `end` value is `86400000` (24 hours * 60 minutes/hour * 60 seconds/minute * 1000 milliseconds/second), so, yes, this is a valid value and represents 6pm.